src/argsPass.js

```diff
@@ -0,0 +1,39 @@
+import { useWith, curry, compose } from 'ramda';
+
+import list from './list';
+import isTruthy from './isTruthy';
+
+/**
+ * Takes a combining predicate and a list of functions and returns a function which will map the
+ * arguments it receives to the list of functions and returns the result of passing the values
```

Comment:
Although this is long, it describes what the functions does. So I'm fine with this.

Reply:
It seems to me that the description here and on typescript signature is not the same. Can we unify them pls ?